heap/max_heap.py

```diff
@@ -10,8 +10,19 @@ def insert(self, value):
 
 
   def delete(self):
-      self.storage.pop(0)
-      return self._bubble_up(len(self.storage) -1)
+      last_index =  self.storage[len(self.storage) - 1]
+      # last index will be 1
+      # this swaps 1 and 999
+      self.storage[0], self.storage[len(self.storage) - 1] = self.storage[len(self.storage) - 1],  self.storage[0]
```

Comment:
so for example: if above, you set `last_index = len(self.storage) - 1`, then you can write this line as

 self.storage[0], self.storage[last_index] = self.storage[last_index],  self.storage[0]